ckanext/security/controllers.py

```diff
@@ -111,8 +111,8 @@ def login(self):
                 # Increment the throttle counter if the login failed.
                 throttle.increment()
 
-            if locked_out or invalid_login:
-                log.info('login failed')
+            if invalid_login or (invalid_login and locked_out):
```

Comment:
Unless I'm missing something, the second branch of the conditional is a subset of the first and therefore unreachable.